RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateReco2D.h

```diff
@@ -0,0 +1,56 @@
+//
+//  SiPixelTemplateReco2D.cc (Version 2.20)
+//  Updated to work with the 2D template generation code
+//  2.10 - Add y-lorentz drift to estimate starting point [for FPix]
+//  2.10 - Remove >1 pixel requirement
+//  2.20 - Fix major bug, change chi2 scan to 9x5 [YxX]
+
+
+//
+//
+//  Created by Morris Swartz on 7/13/17.
+//
+//
+
+#ifndef SiPixelTemplateReco2D_h
```

Comment:
Please use an header guard with the full path

Reply:
I'll suggest this to Morris.  (This is just an arbitrary string, after all.)  But there better be no other SiPixelTemplateReco2D code in CMSSW.